pkgs/os-specific/linux/unscd/default.nix

```
@@ -0,0 +1,76 @@
+{ fetchurl, fetchpatch, stdenv, systemd, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "unscd";
+  version = "0.54";
+
+  src = fetchurl {
+    url = "https://busybox.net/~vda/${pname}/nscd-${version}.c";
```

```suggestion
    url = "https://busybox.net/~vda/unscd/nscd-${version}.c";
```
easily copyable links are very nice.